### PR TITLE
Adding prerun hook to lowercase application names

### LINKED ISSUE #209: draft up fails with capitals in application name
When deploying using `draft up` with a CamelCase name (or just a name with capitals in general), draft fails with the following:

```
$ draft up
--> Building Dockerfile
Could not build image from build context: Error response from daemon: invalid reference format: repository name must be lowercase
```

We either need to cast the name to lowercase to work around this limitation of docker image name formats or enforce it.

### PR DESCRIPTION
**Pull request by bnookala**
Fixes #209! 

:wave: Hi @bacongobbler! I thought this would be a quick and easy fix to solve issue #209. Let me know what you think!

### CODE REVIEW

**Comment by bnookala** [comment]
@bacongobbler was wondering if there was anything else that needed to be done for this branch too? :)

**Comment by michelleN** [comment]
@bnookala Sorry for the delay on this PR. I'd love to get this merged in. Would you mind rebasing on master and resolving conflicts? Thank you!

**Comment by bnookala** [comment]
@michelleN can do!

**Comment by bnookala** [comment]
@michelleN rebased! *fingers crossed*

File: cmd/draft/create.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package main
+	package main
```
Comment (bacongobbler) [inline cmd/draft/create.go]: mind moving this back? `package` doesn't need to be indented. `go fmt ./...` oughta do the trick. :)

Comment (bnookala) [reply cmd/draft/create.go]: @bacongobbler d'oh; probably leftover from a merge - should be fixed now.

**Pull request merged**